lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -859,6 +859,37 @@ function dosomething_signup_set_presignup_message($title) {
 }
 
 /**
+* Returns total number of Signups based on supplied set of filters. 
+*
+* @param array $filters
+* @return int
+*
+*/
+function dosomething_signup_get_signup_total_by_filters(array $filters = []) {
```

Comment:
So weird! I deleted it and pushed it up then re-added and pushed it up and still doesn't show php highlighting :-/